packages/web3-hooks/base/src/index.ts

```diff
@@ -97,3 +97,4 @@ export * from './useWeb3State.js'
 export * from './useWeb3UI.js'
 export * from './useZeroAddress.js'
 export * from './usePageableAsync.js'
+export * from './ScopedDomains/index.js'
```

Comment:
Move to `@masknet/shared-ui`.

Reply:
You mean `@masknet/shared-base-ui`? This hook is not about UI but address/domain management.